docs/development/i18n.rst

```diff
@@ -64,6 +59,19 @@ files into the corresponding ``*.j2`` file and committed to the
 SecureDrop repository. They are then installed when configuring Tails
 with the ``tasks/create_desktop_shortcuts.yml`` tasks.
 
+We don't expect translators to deal with all these files directly, of
```

Comment:
nit: I recommend getting rid of "of course". It feels out of place because it's not obvious yet what translators are expected to do if you're a developer being introduced to the translation system for the first time.